src/vips-emscripten.cpp

```diff
@@ -433,6 +434,8 @@ EMSCRIPTEN_BINDINGS(my_module) {
     function("config", optional_override([]() {
                  return vips::replace_all(VIPS_CONFIG, ", ", "\n");
              }));
+    function("blockUntrusted", &vips::block_untrusted_set);
+    function("operationBlock", &vips::operation_block_set);
```

Comment:
Perhaps we could use `optional_override` to avoid the need of a separate `init.h` file?
```suggestion
    function("blockUntrusted", optional_override([](bool state) {
                 vips_block_untrusted_set(state ? 1 : 0);
             }));
    function("operationBlock",
             optional_override([](const std::string &name, bool state) {
                 vips_operation_block_set(name.c_str(), state ? 1 : 0);
             }));
```

Reply:
Sure, moved in https://github.com/kleisauke/wasm-vips/pull/24/commits/97f3648831a2deb1f7e67f48bdd935661ff93a67